gettsim/parameters/eink_st.yaml

```diff
@@ -178,6 +178,28 @@ eink_st_tarif:
       upper_threshold: 57051
     3:
       upper_threshold: 270501
+  2021-01-01:
+    deviation_from: previous
+    reference: Art. 1 G. v. 01.12.2020 BGBl. I S. 2616
```

Comment:
A nitpick on consistency: In #251 you added a note for the same law reference (2. FamEntLastG again here), and a full stop after the page number. I think mentioning "Zweites Familienentlastungsgesetz" is not necessary and rather a reference, not a note. What do we put in the note key? calculations?

Reply:
I added a note because is has been done before like that: https://github.com/iza-institute-of-labor-economics/gettsim/blob/1f11c71fc0f63f6160fbae065bb7bd73c6a6aaa3/gettsim/parameters/eink_st.yaml#L114-L129

So `note` is the right key in my view and I could obviously add those here.